src/app/demo/demo.component.html

```diff
@@ -1,51 +1,40 @@
 <div class="container">
-  <div class="nlsg-c-form-element__toggle-wrapper">
+  <div class="toggle-wrapper">
     <label>Freeze card</label>
-    <label class="nlsg-c-form-element__toggle-wrapper__toggle">
-      <input type="checkbox" (click)="open(content)" (change)="firstModel = !firstModel;">
-      <span class="nlsg-c-form-element__toggle-wrapper__toggle__slider round"></span>
+    <label class="toggle-wrapper__toggle">
+      <input type="checkbox" (click)="open(content)" (change)="openModel = !openModel;">
```

Comment:
Check for the typo here, it's supposed to be openModal

Reply:
done